arbor/include/arbor/mechanism.hpp

```diff
@@ -55,8 +57,8 @@ class mechanism {
 
     // Forward to interface methods
     void initialize()     { ppack_.vec_t = *time_ptr_ptr; iface_.init_mechanism(&ppack_); }
```

Comment:
It's probably not as useful, but I would suggest -- just for completeness -- to instrument all methods here.

Reply:
The reason they are not is because they're included in another section of the profiler. I agree though, I'll see what I can move around.